packages/orchestrator-ui-components/src/components/WfoInlineNoteEdit/WfoSubscriptionNoteEdit.tsx

```diff
@@ -1,35 +1,32 @@
 import type { FC } from 'react';
 import React from 'react';
 
-import { ApiResult, SubscriptionListResponse, UseQuery } from '@/rtk';
 import { useStartProcessMutation } from '@/rtk/endpoints/forms';
 import { useUpdateSubscriptionNoteOptimisticMutation } from '@/rtk/endpoints/subscriptionListMutation';
 import { Subscription } from '@/types';
 import { INVISIBLE_CHARACTER } from '@/utils';
 
 import { WfoInlineEdit } from '../WfoInlineEdit';
+import { SubscriptionListItem } from '../WfoSubscriptionsList';
 
 interface WfoSubscriptionNoteEditProps {
     subscriptionId: Subscription['subscriptionId'];
     onlyShowOnHover?: boolean;
     queryVariables: Record<string, unknown>;
-    useQuery: UseQuery<SubscriptionListResponse, Subscription>;
+    endpointName: string | undefined;
+    data: SubscriptionListItem[];
 }
 
 export const WfoSubscriptionNoteEdit: FC<WfoSubscriptionNoteEditProps> = ({
     subscriptionId,
     onlyShowOnHover = false,
     queryVariables,
-    useQuery,
+    endpointName,
+    data,
 }) => {
-    const { selectedItem } = useQuery(queryVariables, {
-        selectFromResult: (result: ApiResult<SubscriptionListResponse>) => ({
-            selectedItem: result?.data?.subscriptions?.find(
-                (sub) => sub.subscriptionId === subscriptionId,
-            ),
-        }),
-    });
-    const endpointName = useQuery().endpointName;
+    const selectedItem = data.find(
+        (sub) => sub.subscriptionId === subscriptionId,
+    );
```

Comment:
Can't we just pass the current row into this component? Now we pass all data and we need to perform a `find` to retrieve it again

Reply:
yup 🤦‍♂️